src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,88 @@
+REM
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
+REM %3: Response files location
+
+@echo off
+setlocal
+
+set BridgeHost=
+pushd %2
+pushd bin\wcf\tools\Bridge
+if [%ERRORLEVEL%]==[0] (
+    call Bridge.exe -stop
+    popd
+)
+
+#REM remove bin directory to ensure a rebuild
+call git clean -xdf
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while cleanning Git.
+    exit /b
+  )
+
+REM sync
+git checkout master
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while checking out master.
+    exit /b
+  )
+git pull
+if NOT [%ERRORLEVEL%]==[0] (
+    echo ERROR: An error occurred while syncing master.
+    exit /b
```

Comment:
Actually, no, you shouldn't be doing this; you never end up popd-ing. You should have a label at the end of the script (e.g., :done) where you popd then exit /b %captured_errorlevel%